src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -500,15 +519,14 @@ const formConfig = {
           title: 'Applicant information',
           path: 'applicant-info',
           uiSchema: {
-            ...titleUI('Applicant name and date of birth', () => (
+            ...titleUI('Applicant name and date of birth', ({ formData }) => (
+              // Prefill message conditionally displays based on `certifierRole`
               <>
-                Enter the information for any applicants you want to enroll in
-                CHAMPVA benefits.
-                <br />
-                <br />
-                {`You can add up to ${MAX_APPLICANTS} applicants in a single application. If you 
-              need to add more than ${MAX_APPLICANTS} applicants, you'll need to submit a 
-              separate application for them.`}
+                <p>
+                  Enter the information for any applicants you want to enroll in
+                  CHAMPVA benefits.
+                </p>
+                {CustomPrefillMessage(formData, 'applicant')}
```

Comment:
Updating copy + adding the custom prefill message for when the signer is also an applicant. This lets the users know when we've duplicated the signer info for them so they don't have to re-type.